### PR TITLE
Update version command to get version of particular component

### PR DESCRIPTION
**Pull request by pratap0007**
This patch adds flag `component` to get the
version of a particular component and also
version command show available component versions

### CODE REVIEW

File: pkg/version.go [hunk 1]
```diff
@@ -16,48 +16,127 @@ import (
...
-					fmt.Fprintln(ioStreams.Out, v.Pac)
-				case "tkn":
-					fmt.Fprintln(ioStreams.Out, v.Tkn)
-				case "opc":
-					fmt.Fprintln(ioStreams.Out, v.Opc)
-				default:
-					return fmt.Errorf("unknown component: %v", args[1])
-				}
+			if args[0] == "pac" {
+				fmt.Fprintf(cmd.OutOrStdout(), "%s\n", v.Opc)
```
Comment (chmouel) [inline pkg/version.go:53]: is that intended?

Comment (chmouel) [reply pkg/version.go:53]: you should be able to get the pac version from runtime as cli

**Review by chmouel — COMMENTED** [review]
it gets complicated enough that we need to start having unittest now, any chance you can add some (and a way to run them via makefile and actions CI)?

**Comment by pratap0007** [comment]
closing this PR as there is better implementation in this PR https://github.com/openshift-pipelines/opc/pull/161

**Pull request closed without merging**